deployment.Rmd

```diff
@@ -58,6 +58,22 @@ library(sparklyr)
 sc <- spark_connect(master = "yarn-client")
 ```
 
+or for more customized configuration:
+
+```{r}
+library(sparklyr)
+sc <- spark_connect(master="yarn",
+                    config = list(
```

Comment:
Let's say we'll have one config.yml with 3 profiles. How can then I refine the specific profile?